internal-scripts/test-debs

```diff
@@ -280,6 +280,20 @@ if [[ "$VARIANT" = jemalloc ]]; then
     run c++ "-I$INCLUDE_DIR" -Wall /system/resources/test-env/jemalloc-cxx.cpp -o /tmp/jemalloc_cxx
 fi
 
+MAJOR_VERSION=`cut -d '.' -f 1 <<< "$RUBY_PACKAGE_VERSION"`
```

Comment:
To keep shellcheck happy, you should use `$(...)` instead of backticks.

Reply:
Ditto for test-rpms.

Reply:
Nice catch, tweaked at d5a1653!

Reply:
Nice catch, tweaked at d5a1653!